pyodide_build/mkpkg.py

```diff
@@ -79,12 +80,17 @@ def make_parser(parser):
     parser.add_argument(
         'package', type=str, nargs=1,
         help="The package name on PyPI")
+    parser.add_argument(
+        '--version', type=str, nargs=1, default=None,
+        help="Package version string, "
+             "e.g. v1.2.1 (defaults to latest stable release)")
     return parser
 
 
 def main(args):
     package = args.package[0]
-    make_package(package)
+    version = args.version[0] if args.version else None
+    make_package(package, version)
```

Comment:
```suggestion
    make_package(package, args.version)
```
if you remove the `nargs` argument above.